js-miniapp-sample/README.md

```diff
@@ -0,0 +1,133 @@
+# miniapp-build-common
```

Comment:
I would recommend changing this to `js-miniapp-sample` to reflect the new root folder